src/org/traccar/processing/ComputedAttributesHandler.java

```diff
@@ -108,7 +108,22 @@ protected Position handlePosition(Position position) {
                                 position.getAttributes().put(attribute.getAttribute(), (Boolean) result);
                                 break;
                             default:
-                                position.getAttributes().put(attribute.getAttribute(), result.toString());
+                                String attributeValue = null;
+                                String attributeName = attribute.getAttribute();
+                                if (attributeName.equals("alarm")
+                                        || attributeName.equals("event")) {
```

Comment:
Why do we need to handle event here? Event doesn't have any special significance.

Also, you have to use KEY_* from position instead of hardcoded strings.

Reply:
If a user defines multiple "Event" then they should be all evaluated like Alarms, right?

I forget to use the KEY_* constants, sorry.

Reply:
Format for events field is not defined. It can be string, number or anything else. We should not convert it to string and append something to it.

Reply:
Ok, will remove that